<!--?xml version=1.2 ?-->
<launch>
  <node pkg="opencart_refbox" name="order_node" type="order_node" output="screen">
    <param name="order_topic" value="/orderinfo" />
    <param name="refbox_host" value="localhost" />
    <param name="refbox_port" value="4444" />
    <param name="message_path" value="$(env HOME)/edge/rcll-refbox/src/msgs/" />
  </node>

</launch>
